Source/EPUBCore/FRBook.swift

```diff
@@ -40,18 +40,18 @@ open class FRBook: NSObject {
     // http://www.idpf.org/epub/301/spec/epub-mediaoverlays.html#sec-package-metadata
 
     var duration: String? {
-        return metadata.findMetaByProperty("media:duration")
+        return metadata.find(byProperty: "media:duration")?.value
     }
 
     var activeClass: String {
-        guard let className = metadata.findMetaByProperty("media:active-class") else {
+        guard let className = metadata.find(byProperty: "media:active-class")?.value else {
             return "epub-media-overlay-active"
         }
         return className
     }
 
     var playbackActiveClass: String {
-        guard let className = metadata.findMetaByProperty("media:playback-active-class") else {
+        guard let className = metadata.find(byProperty: "media:playback-active-class")?.value else {
```

Comment:
Same here and in the others that use `.value`

Reply:
The same as above